### PR TITLE
New JSON webhook type, new doc

### PR DESCRIPTION
**Pull request by romain-growthbook**
This PR introduces a new `JSON` event webhook type. This new type is used to map to the new event webhook. payloads.

Existing `raw` webhooks are then considered legacy and frozen:
* No new `raw` webhook can be created
* Existing `raw` webhook can be edited but their event list and type are also frozen and cannot be changed.

This makes it possible to switch to the new type of webhooks without having to migrate any data!

Also, the doc is updated to reflect the new webhook and their payload. See screenshot below.

### Screenshots

**App setup**

Screenshot 2024-08-23 at 9 39 45 AM

Screenshot 2024-08-23 at 9 39 33 AM

**Doc**

Screenshot 2024-09-06 at 11 56 18 AM

Screenshot 2024-09-05 at 10 25 55 AM

### CODE REVIEW

**Review by royalfig — COMMENTED** [review]
Docs lgtm. I like the display of the webhook payloads.

File: docs/src/partials/event-webhook/_event-webhook-list.md
Comment (royalfig) [inline docs/src/partials/event-webhook/_event-webhook-list.md:1]: Would it be less repetitive/clearer to format the table like this?

| Event Name | Trigger |
| --- | --- |
| **[feature.created](#featurecreated)** | Feature created |
| **[experiment.warning](#experimentwarning)** | Warning condition detected on experiment |
| **[user.login](#userlogin)** | User logs in |

Comment (royalfig) [reply docs/src/partials/event-webhook/_event-webhook-list.md:1]: @romain-growthbook I know you merged this, but a thought for the next iteration

**Review by jdorn — APPROVED** [review]

**Comment by jdorn** [comment]
Approved.  Just want to make sure you've manually tested this end-to-end before merging.  Specifically, a webhook of type `raw` should continue to fire (backwards compatible) and a new webhook of type `json` should also fire as expected.

**Pull request merged**